generator/app.go

```diff
@@ -233,12 +233,16 @@ func (c *Context) Validate() []error {
 			for prefix, persons := range group.Leadership.PrefixToPersonMap() {
 				for _, person := range persons {
 					if val, ok := people[person.GitHub]; ok {
-						if val.Name != person.Name || val.Company != person.Company {
+						if val.Name != person.Name || (prefix != "emeritus_lead" && val.Company != person.Company) {
 							errors = append(errors, fmt.Errorf("%s: %ss: expected person: %v, got: %v", group.Dir, prefix, val, person))
 						}
-					} else {
+					} else if prefix != "emeritus_lead" {
```

Comment:
With this change, we end up loosing the following validation but I'm not sure how to check this without introducing additional complexity. Also not sure if we care enough about this case? :confused: 

If someone is an `emeritus_lead` + has some other role (chair, TL, etc) as well, we won't be able to validate if their name in the `emeritus_leads` section is the same as that in other places.

Reply:
I don't think we have to worry about validation in `emeritus_lead`. It's a polite nod, and attribution -- not something that we depend on.